src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown

```diff
@@ -104,7 +104,7 @@ Sign In or Sign Up| This is used when the customer may not have an account or re
 
 #### <a name="landing-page-verification"></a>Verification
 
-If your application activation requires pre-requisites (e.g. a contract or other manual set up is required), you may introduce an additional verification step. Once all pre-requisites are complete and prior to the user activating the application, you will supply the administrator with a verification code. Once the user connects from the SAP Concur App Center, your landing page will prompt the user for a verification code; the user must provide a verification code to proceed. In the remainder of this guide, this will be referred to as a “Verification” app. This page should clearly articulate the next steps and should provide the user the option to:
+This is the recommended option for the majority of Enterprise applications since the partner will likely have had contract negotiations with the customer outside of the Concur connection flow.  Once the contract is finalized with the customer, they will have the value to copy and paste into the Verfication field of your Landing Page. If your application activation fits this description and requires pre-requisites (e.g. a contract or other manual set up is required), you may introduce an additional verification step. Once all pre-requisites are complete and prior to the user activating the application, you will supply the administrator with a verification code. Once the user connects from the SAP Concur App Center, your landing page will prompt the user for a verification code; the user must provide a verification code to proceed. In the remainder of this guide, this will be referred to as a “Verification” app. This page should clearly articulate the next steps and should provide the user the option to:
```

Comment:
Suggested text: "This is the recommended option for paid Enterprise applications. Once the contract has been signed by the customer, you will provide the customer with a verification code to enter on your landing page. In the remainder of this guide..."